src/components/common/Spinner.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+
+const SpinnerWrapper = styled.div`
+  width: 28px;
+  height: 28px;
+  border: 3px solid #fff;
+  border-top-color: ${({ theme }) => theme.colors.semantic.kakaoBrown};
+  border-radius: 50%;
+  animation: spin 0.8s linear infinite;
+
+  @keyframes spin {
+    to {
+      transform: rotate(360deg);
+    }
+  }
+`;
+
+export default function Spinner() {
+  return <SpinnerWrapper />;
+}
```

Comment:
```suggestion
const spinner = <SpinnerWrapper />
```
잘 생각해보면 상태에 의존적인 부분이 없어요.
이렇게 미리 node로 만들어서 삽입해줘도 된답니다 ㅎㅎ